index.js

```diff
@@ -253,7 +253,7 @@ export function createAnimatableComponent(component) {
     componentDidMount() {
       const { animation, duration, delay, onAnimationBegin, onAnimationEnd } = this.props;
       if (animation) {
```

Comment:
Not very easily, as we need some logic to run, even when animation is `none`.
Please see `onAnimationBegin()` on line 273, and the following lines.